util/HalideTraceViz.cpp

```diff
@@ -898,7 +905,7 @@ struct Surface {
         return image[frame_size.x * y + x];
     }
 
-    void draw_text(const std::string &text, const Point &pos, uint32_t color) {
+    void draw_text(const std::string &text, const Point &pos, uint32_t color, float h_scale = 1.5f) {
```

Comment:
1.5f seems like a strange default value - is it ever called with the default value?